include/mbedtls/cipher_internal.h

```diff
@@ -64,6 +64,14 @@ struct mbedtls_cipher_base_t
                      unsigned char *output );
 #endif
 
+#if defined(MBEDTLS_CIPHER_MODE_OFB)
+    /** Encrypt using OFB (Full length) */
+    int (*ofb_func)( void *ctx, size_t length, size_t *iv_off,
```

Comment:
Note: this is technically an ABI change, but on a structure that should never be held directly by the application (as evidenced by the `internal` in the header name), so can be ignored.